docs/running-node.md

```diff
@@ -342,3 +342,85 @@ As a general guidance you may consider to use only the peers you own and/or trus
   * *Notes*
     * `cosmovisor` service should be restarted on any configuration changes
     * in case of any IP filtering firewall rules they should be updated as well
+
+## Adding new full node to existing network
+Possible options when adding Validator, Observer, Sentry or Seed nodes to existing DCL network:
+> **_NOTE:_** Extra node specific configuration (out of scope of this section) may apply depending on node type or infrastructure
+1) Catchup from genesis:
+  - Prerequisites:
+    - All binary versions used for upgrading (using `cosmovisor`) existing network up to current state must be available
+  - Steps:
+    - Add a node with a binary version as was at the genesis state
+    - Let the nodes catch-up and play all updates/migrations
+  
+  - Pros:
+    - The new node contains all the blocks from blockchain (full history)
+
+  - Cons:
+    - Can take quite a lot of time (depends on blockchain size)
+    - Probably error-prone (if at least one migration has bug catchup fails)
+
+2) Take the data from another node
+- Prerequisites:
+  - Access to a machine running an up-to-date node to copy data from
+
+- Steps:
+  - Stop running an up-to-date node (to have consistent data), and copy the data folder
+  - Put that data to the new node's data folder (can be automated)
+  - Run the new node
+
+- Pros:
+  - No specific node configuration needed (just start the new node)
+
+- Cons:
+  - Needs an access to another machine (may not be possible to everyone)
+  - Node that data is copied from should be stopped (downtime)
+  - Probably error prone due to manual operations 
+
+* References:
+  - https://docs.cosmos.network/master/core/upgrade.html#syncing-a-full-node-to-an-upgraded-blockchain
+3) State sync (Recommended)
```

Comment:
I would start with Option 3 (the recommended one), then describe Option 2, and only then Option 1.